src/js/state/Queries/initial.ts

```diff
@@ -6,24 +6,104 @@ const init = (): QueriesState => ({
   items: [
     {
       id: "1",
-      name: "Count by Path",
-      zql: "* | count() by _path",
-      description: "",
-      tags: []
+      name: "Activity Overview",
+      value: "count() by _path | sort -r",
+      description:
+        "This query shows a list of all Zeek streams in the data set, with a count of associated records",
+      tags: ["zeek", "initial exploration"]
```

Comment:
hmm, I wonder if we should allow tags with spaces in them.

Reply:
Good question, if I am the user then I'd prefer to be able to use multiple words with spaces. Looks cleaner. But it does complicate how we detect start and end when we go to edit/create